docs/reference/search/async-search.asciidoc

```diff
@@ -315,3 +315,8 @@ Otherwise, the saved search results are deleted.
 DELETE /_async_search/FmRldE8zREVEUzA2ZVpUeGs2ejJFUFEaMkZ5QTVrSTZSaVN3WlNFVmtlWHJsdzoxMDc=
 --------------------------------------------------
 // TEST[continued s/FmRldE8zREVEUzA2ZVpUeGs2ejJFUFEaMkZ5QTVrSTZSaVN3WlNFVmtlWHJsdzoxMDc=/\${body.id}/]
+
+If the {es} {security-features} are enabled, the deletion of a specific async
+search is restricted to:
+  * The authenticated user that submitted the original search request.
```

Comment:
I think it's worth finding a better way to express how we identify the identify a user.
But, we'll update the docs afterwards, no need to hold this PR in the meantime. It's on me to follow-up.

Reply:
++, thanks